code/modules/projectiles/projectile.dm

```diff
@@ -731,6 +731,9 @@ So if we are on the 32th absolute pixel coordinate we are on tile 1, but if we a
 
 
 /mob/living/carbon/xenomorph/projectile_hit(obj/projectile/proj, cardinal_move, uncrossing)
+	if(SEND_SIGNAL(src, COMSIG_XENO_PROJECTILE_HIT, proj, cardinal_move, uncrossing) & COMPONENT_EVASION_DODGE)
+		return FALSE
```

Comment:
COMPONENT_EVASION_DODGE is not a generic return flag
make the naming more generic